build.sbt

```diff
@@ -564,7 +564,10 @@ lazy val http =
     .settings(buildSettings)
     .settings(
       name := "airframe-http",
-      description := "REST API Framework"
+      description := "REST API Framework",
+      libraryDependencies ++= Seq(
+        "com.twitter" %% "finagle-base-http"   % FINAGLE_VERSION,
```

Comment:
airframe-http should not depend on finagle to minimize the dependencies. That's why we are isolating airframe-http-finagle as another project.